tgui/packages/tgui/components/Box.tsx

```diff
@@ -256,7 +257,7 @@ export const computeBoxClassName = (props: BoxProps) => {
   ]);
 };
 
-export const Box = (props: BoxProps) => {
+export const Box: SFC<BoxProps> = forwardRef((props: BoxProps, ref) => {
```

Comment:
This will hit performance very terribly, please don't do it.

Reply:
I think keybindings menu needed this so that we could get refs of the sections to teleport to, I believe because it used `Stack.Item` to get a ref to. Maybe I can use wrapper divs instead.

Reply:
Just use normal divs, if possible, no need for wrappers.

Reply:
I meant wrapping the content with the div, ya.

Reply:
I limited it to Stack.Item only, which uses div.

Reply:
I fixed this to `innerRef`